README.md

```diff
@@ -79,19 +79,22 @@ this addon directly.
 treeForAddon(tree) {
   let addon = this.addons.find(addon => addon.name === 'ember-cli-babel'); // find your babel addon
 
-  let options = addon.buildBabelOptions({
-    'ember-cli-babel'
-  })
-
-  return addon.transpileTree(tree, {
+  // optionally build up the configuration
+  let config = {
     'babel': {
       // any babel specific options
      },
 
     'ember-cli-babel': {
       // any ember-cli-babel options
     }
-  });
+  };
+
+  // create the babel options to use elsewhere based on the config above
+  let options = addon.buildBabelOptions(config)
```

Comment:
I'm mostly trying to show that you can build options and do _something_ with it yourself (e.g. call your own `require('babel-core').transform` or something.  Not really that the two examples have anything to do with each other.